types/node/globals.d.ts

```diff
@@ -1,381 +1,386 @@
-// Declare "static" methods in Error
-interface ErrorConstructor {
-    /** Create .stack property on a target object */
-    captureStackTrace(targetObject: object, constructorOpt?: Function): void;
+export {} // Make this a module
```

Comment:
This makes `import {} from "node/globals"` legal 😕 That’s why my `_Request` et al. types are accessible. There’s no way to hide them without declaring a module.

Reply:
I guess you learn something new every day.  When I [messed this up last time](https://github.com/thw0rted/DefinitelyTyped/blob/node-add-responseinit/types/node/dom-events.d.ts), nobody seems to have noticed.  Does that need to change as well?

Would it be preferable to move these declarations out of `globals.d.ts` and into some other module context?  I was mostly trying to avoid exposing the fictional namespace so that library authors wouldn't come to depend on that implementation detail, but of course I'd defer to the team about which hazard is more important to work around.

Reply:
Well, if nobody noticed the last one, probably nobody will notice this. I guess it’s fine 😄 

If we had a type reference directive condition, we could use it to hide everything from imports while keeping things accessible to type reference directives:


```json
{
  "exports": {
    "./*": {
      "types@>=5.5": {
        "typereference": "./*",
        "default": null
      },
      "default": "./*"
    }
  }
}
```

🤔